sdk/resourcemanager/Azure.ResourceManager.Core/tests/Scenario/PreDefinedTagsContainerTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.s
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Core.Tests
+{
+    public class PreDefinedTagsContainerTests : ResourceManagerTestBase
+    {
+        public PreDefinedTagsContainerTests(bool isAsync)
+            : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        [OneTimeTearDown]
+        protected void GlobalTagCleanup()
+        {
+            var container = Client.DefaultSubscription.GetPredefinedTags();
+            var operation = Client.GetTagOperations();
+            var listResult = container.List().Where(x => x.Details.TagName.StartsWith("tagName"));
+            foreach (var item in listResult)
+            {
+                operation.Delete(item.Details.TagName);
```

Comment:
shouldn't we be able to do item.Delete?

also use async

Reply:
Done